packages/xrpl/src/sugar/balances.ts

```diff
@@ -1,123 +1,21 @@
-import flatMap from 'lodash/flatMap'
+import { AccountLinesTrustline } from '../models'
 
-import type { Balance, Client } from '..'
-import {
-  AccountLinesRequest,
-  AccountLinesTrustline,
-  LedgerIndex,
-  AccountInfoRequest,
-} from '../models'
-import { dropsToXrp } from '../utils'
+export interface Balance {
```

Comment:
You should use https://github.com/XRPLF/xrpl.js/blob/d47a277e25ae1a1ff95c7bd5a35dd133286c4330/packages/xrpl/src/models/common/index.ts#L20

Reply:
Fixed